newIDE/app/src/UI/ScrollView.js

```diff
@@ -47,6 +49,28 @@ export default React.forwardRef<Props, ScrollViewInterface>(
           );
         }
       },
+      /**
+       * Scroll the view to the target position.
+       */
+      scrollToPosition: (y: number) => {
+        const scrollViewElement = scrollView.current;
+        if (!scrollViewElement) return;
+
+        const scrollViewYPosition = scrollViewElement.getBoundingClientRect()
+          .top;
+        scrollViewElement.scrollTop += y - scrollViewYPosition;
```

Comment:
You can maybe put this in common in an util that the `scrollTo` method would use as well.